File: algorithms/01_wage_increase.py
wage = float(input("Digite o valor do salário: "))
percentage = float(input("Digite a porcentagem do aumento: "))

increase = (wage * (percentage / 100))
new_wage = wage + increase

print("O valor do aumento foi de R$%.2f" % increase)
print("Com o aumento o novo salário é de R$%.2f" % new_wage)

File: algorithms/01_wage_increase.c
#include <stdio.h>
int main()
{
    double wage;
    double percentage;

    printf("Digite o valor do salário: ");
    scanf("%lf", &wage);
    printf("Digite a porcentagem do aumento: ");
    scanf("%lf", &percentage);

    double increase = (wage * (percentage / 100));
    double new_wage = wage + increase;

    printf("O valor do aumento foi de R$%.2lf\n", increase);
    printf("Com o aumento o novo salário é de R$%.2lf\n", new_wage);

    return 0;
}